### PR TITLE
Create readme.md

### PR DESCRIPTION
**Pull request by moekappels**
this adds a readme that explains how to use the program after pull request #25 was merged into master.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# ethereum-cache-creator
+## How to use program in the current iteration
+* download the [artifact](https://github.com/internet-sicherheit/ethereum-cache-creator/actions/runs/104782193)
+* unpack zip with a extractor of your choice, I would recommend [7zip](https://www.7-zip.de/).
+* open folder/directory in terminal
+* To run the program: 
+```
+java -jar BloxbergCacheCreator.jar <arguments>
```
Comment (kiview) [inline README.md:8]: Here should come an additional example about calling it with some actual arguments, like:
`java -jar BloxbergCacheCreator.jar --ui=cli --stop=3000`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+```
+The list of additional command line arguments, if required.
+
+indicator | value | defaults
+----------- | ----------------- | --------------
+--ui=gui      | to run with GUI | *no default*
```
Comment (kiview) [inline README.md:14]: Hm, it feels like CLI is the effective default?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# ethereum-cache-creator
+## How to use program in the current iteration
+* download the [artifact](https://github.com/internet-sicherheit/ethereum-cache-creator/actions/runs/104782193)
```
Comment (kiview) [inline README.md:3]:
```suggestion
* Download the [artifact](https://github.com/internet-sicherheit/ethereum-cache-creator/actions/runs/104782193)
```

There should be another link the might give you the most recent artifact, but I  am not sure about this.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+* unpack zip with a extractor of your choice, I would recommend [7zip](https://www.7-zip.de/).
```
Comment (kiview) [inline README.md:4]:
```suggestion
* Extract the zip file
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+* open folder/directory in terminal
```
Comment (kiview) [inline README.md:5]:
```suggestion
* Open the extracted directory in a terminal
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+* To run the program:
```
Comment (kiview) [inline README.md:6]:
```suggestion
* Execute the program:
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+----------- | ----------------- | --------------
+--ui=gui      | to run with GUI | *no default*
+--url=     | the url of the client | https://core.bloxberg.org
+--filename= | the name of the file | transactions_from_to
+--start=  | first block to extract data from | 0
+--stop=  | last block to extract data from | 1000
+
+**Notes**
+
+- file will appear in folder "output".
```
Comment (kiview) [inline README.md:22]:
```suggestion
- The result file will appear in a newly created directory `output`.
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+- .json will be added automatically to filename.
```
Comment (kiview) [inline README.md:23]:
```suggestion
- A `.json` suffix will be added automatically to the specified filename.
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+- Start and stop need to be numbers between 0 and 6000000.
```
Comment (kiview) [inline README.md:24]: Why under 6000000?

File: README.md (hunk 5 above)
Comment (moekappels) [reply README.md:7]: The artifact itself does not contain a directory. when you press "extract here" there is no directory just the jar file.

Comment (kiview) [reply README.md]: You are right.

So make it
`Open the directory containing the extracted JAR file in a terminal`

**Pull request merged**